requirements.txt

```diff
@@ -1,6 +1,6 @@
-ipaddr
```

Comment:
do we still need that file with your changes?

Reply:
right, actually we can remove it now I think